fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/ConnectingActivity.java

```diff
@@ -19,20 +19,14 @@
 import com.github.dedis.popstellar.utility.error.ErrorUtils;
 import com.github.dedis.popstellar.utility.security.KeyManager;
 import com.tinder.scarlet.WebSocket;
-
-import java.util.ArrayList;
-import java.util.List;
-import java.util.stream.Collectors;
-
-import javax.inject.Inject;
-
 import dagger.hilt.android.AndroidEntryPoint;
 import io.reactivex.disposables.CompositeDisposable;
 import io.reactivex.observables.ConnectableObservable;
+import java.util.*;
```

Comment:
The imports are being reordered everywhere, this is probably a formatting issue.

Reply:
I double checked this multiple time even updating IDE and reinstalling the google format. The result is always the same, seems like they changed import formatting. Can you double check as well?